EXAMPLES.md

```diff
@@ -890,3 +890,117 @@ spec:
 Once it's applied, whoami should be accessible on http://whoami.docker.localhost/
 
 </details>
+
+# Use Kubernetes Gateway API with cert-manager
+
+One can use the new stable kubernetes gateway API provider with automatic TLS certificates delivery (with cert-manager) setting the following _values_:
+
+```yaml
+image:
+  tag: v3.1.0
+
+ingressRoute:
+  dashboard:
+    enabled: true
+
+providers:
+  kubernetesGateway:
+    enabled: true
+
```

Comment:
```suggestion
providers:
  kubernetesGateway:
    enabled: true
```

Reply:
1. We will release in a follow up PR a new release defaulting to v3.1.
2. The dashboard is not related to the subject of this example